--- sesman.ini	2023-04-17 14:12:19.096543177 +0900
+++ /etc/xrdp/sesman.ini	2022-05-06 01:10:13.436961591 +0900
@@ -3,11 +3,11 @@
 [Globals]
 ListenAddress=127.0.0.1
 ListenPort=3350
-EnableUserWindowManager=true
+EnableUserWindowManager=0
 ; Give in relative path to user's home directory
-UserWindowManager=startwm.sh
+UserWindowManager=startubuntu.sh
 ; Give in full path or relative path to /etc/xrdp
-DefaultWindowManager=startwm.sh
+DefaultWindowManager=startubuntu.sh
 ; Give in full path or relative path to /etc/xrdp
 ReconnectScript=reconnectwm.sh
 
@@ -119,7 +119,7 @@
 ; See sesman.ini(5) for the format of this parameter
 #FuseMountName=/run/user/%u/thinclient_drives
 #FuseMountName=/media/thinclient_drives/%U/thinclient_drives
-FuseMountName=thinclient_drives
+FuseMountName=shared-drives
 ; this value allows only the user to acess their own mapped drives.
 ; Make this more permissive (e.g. 022) if required.
 FileUmask=077
