--- xrdp.ini    2023-04-17 14:11:27.509747359 +0900
+++ /etc/xrdp/xrdp.ini  2022-05-06 01:10:13.428961546 +0900
@@ -11,7 +11,7 @@
 ; ALL specified interfaces must be UP when xrdp starts, otherwise xrdp will fail to start
 ;
 ; Examples:
-;   port=3389
+;   port=vsock://-1:3389
 ;   port=unix://./tmp/xrdp.socket
 ;   port=tcp://.:3389                           127.0.0.1:3389
 ;   port=tcp://:3389                            *:3389
@@ -20,7 +20,7 @@
 ;   port=tcp6://:3389                           *:3389
 ;   port=tcp6://{<any ipv6 format addr>}:3389   {FC00:0:0:0:0:0:0:1}:3389
 ;   port=vsock://<cid>:<port>
-port=3389
+port=vsock://-1:3389

 ; 'port' above should be connected to with vsock instead of tcp
 ; use this only with number alone in port above
@@ -41,12 +41,12 @@

 ; security layer can be 'tls', 'rdp' or 'negotiate'
 ; for client compatible layer
-security_layer=negotiate
+security_layer=rdp

 ; minimum security level allowed for client for classic RDP encryption
 ; use tls_ciphers to configure TLS encryption
 ; can be 'none', 'low', 'medium', 'high', 'fips'
-crypt_level=high
+crypt_level=none

 ; X.509 certificate and private key
 ; openssl req -x509 -newkey rsa:2048 -nodes -keyout key.pem -out cert.pem -days 365
@@ -80,7 +80,7 @@
 allow_channels=true
 allow_multimon=true
 bitmap_cache=true
-bitmap_compression=true
+bitmap_compression=false
 bulk_compression=true
 #hidelogwindow=true
 max_bpp=32